src/components/common/Spinner.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Spinner = (props) => {
+  const { isLoading } = props;
+  let style;
+  if (!isLoading) {
+    style = { display: 'none' };
+  } else {
+    style = { display: 'block' };
+  }
+  return (
+    <div style={style} className="spinner-overlay">
+      <i className="fas fa-spinner" />
```

Comment:
you can just add the class `fa-spin` here and remove the css for animating this

Reply:
Thanks. But I want to increase the speed of the spinner.